plugins/Toolbox/src/Toolbox.py

```diff
@@ -234,13 +234,16 @@ def browsePackages(self) -> None:
         # Apply enabled/disabled state to installed plugins
         self.enabledChanged.emit()
 
-    def _createDialog(self, qml_name: str) -> Optional[QObject]:
+    def _createDialog(self, qml_name: str) -> QObject:
```

Comment:
It can return `None`, so it should indicate `Optional`

Reply:
Fixed in ac0192f